lib/validations/storage_root_to_db.rb

```diff
@@ -0,0 +1,29 @@
+class StorageRootToDB
```

Comment:
I know this is WIP;  just noting that we thought `lib/audit` was a better directory name at our meeting this afternoon 2017-10-3